### DISCUSSION TITLE
NGINX is not installed and also remaning commands not working in putty please solve this

### DISCUSSION THREAD

**Discussion by NARENDRA935** [learndevopsonline/learndevopsonline#1683] [Q&A] [answered]
<img width="930" alt="Screenshot 2023-12-20 190105" src="https://github.com/learndevopsonline/learndevopsonline/assets/153573686/65c73579-41fe-42ac-b599-1fbc63930a7a">

**Comment by sreedharm07** [comment]
ping your ip

**Reply by NARENDRA935** [reply]
ip address 3.95.1.23

**Reply by NARENDRA935** [reply]
please check once

**Reply by sreedharm07** [reply]
internet is not comming to your instances
please attach internet gateway and also check security groups that attached to instance 

or just delete that instance and create new instance with proper steps by following viedio

**Reply by NARENDRA935** [reply, marked as answer]
okay tq
